src/app/pages/flex-page/flex-page.scss

```diff
@@ -0,0 +1,4 @@
+
+.page.flex-page {
+  overflow: visible;
```

Comment:
the overflow:hidden on the page was hiding the dropdown menu if there was no following content